app/models/labware_creators/baited_plate.rb

```diff
@@ -25,12 +25,13 @@ def plate
     end
 
     def bait_library_layout_preview
-      @bait_library_layout_preview ||= api.bait_library_layout.preview!(plate: parent_uuid, user: user_uuid).layout
+      @bait_library_layout_preview ||=
+        Sequencescape::Api::V2::BaitLibraryLayout.preview(plate_uuid: parent_uuid, user_uuid: user_uuid).first.layout
```

Comment:
Should preview be returning a list (having to do .first) or is that a quirk of JSON API resources?

Reply:
I think it's a quirk of the client library here.  I tried to stop it from happening, but it seems quite intent.  I decided it wasn't worth spending any more time on and just accept you're going to get a list.